pkg/services/dashboards/database/database.go

```diff
@@ -1007,5 +1007,35 @@ func (d *DashboardStore) FindDashboards(ctx context.Context, query *models.FindP
 		return nil, err
 	}
 
+	if query.CountDB {
```

Comment:
definitely not the way to do it since we want to count the real dashboard number instead of the ones return in the response.... :D